public/stylesheets/board.css

```diff
@@ -1208,8 +1208,19 @@ div.table_wrap > .clock .bar span {
   display: block;
   height: 4px;
   width: 0px;
+}
+div.table_wrap > .clock .bar.green span {
   background: #759900;
 }
+div.table_wrap > .clock .bar.yellow span {
+  background: #ffdd77;
+}
+div.table_wrap > .clock .bar.orange span {
+  background: #f2a463;
+}
+div.table_wrap > .clock .bar.red span {
+  background: #d98d8d;
+}
```

Comment:
Light gold instead? The current gold is too eye-catching, it's even more alerting than red.